### PR TITLE
Change content of error messages when letter job can't be cancelled.

### PR DESCRIPTION
**Pull request by CrystalPea**
This content was first done without help of content designer so now it is improved with the help of one. New content is more in line with the content in the admin app and more human-readable and following GOV.UK standard.

### CODE REVIEW

File: app/dao/jobs_dao.py [hunk 1]
```diff
@@ -177,13 +177,13 @@ def can_letter_job_be_cancelled(job):
...
-        return False, "This job is still being processed. Wait a couple of minutes and try again."
+        return False, "We are still processing these letters, please try again in a minute."
     count_cancellable_notifications = len([
         n for n in notifications if n.status in CANCELLABLE_JOB_LETTER_STATUSES
     ])
     if count_cancellable_notifications != job.notification_count or not letter_can_be_cancelled(
         NOTIFICATION_CREATED, job.created_at
     ):
-        return False, "Sorry, it's too late, letters have already been sent."
+        return False, "It's too late to cancel sending, these letters have already been sent."
```
Comment (quis) [inline app/dao/jobs_dao.py]:
```suggestion
        return False, "It’s too late to cancel sending, these letters have already been sent."
```

File: tests/app/dao/test_jobs_dao.py [hunk 2]
```diff
@@ -342,7 +342,7 @@ def test_can_letter_job_be_cancelled_returns_false_and_error_message_if_notifica
     create_notification(template=job.template, job=job, status='created')
     result, errors = can_letter_job_be_cancelled(job)
     assert not result
-    assert errors == "Sorry, it's too late, letters have already been sent."
+    assert errors == "It's too late to cancel sending, these letters have already been sent."
```
Comment (quis) [inline tests/app/dao/test_jobs_dao.py]:
```suggestion
    assert errors == "It’s too late to cancel sending, these letters have already been sent."
```

File: tests/app/dao/test_jobs_dao.py [hunk 3]
```diff
@@ -355,7 +355,7 @@ def test_can_letter_job_be_cancelled_returns_false_and_error_message_if_letters_
     with freeze_time('2019-06-13 17:32'):
         result, errors = can_letter_job_be_cancelled(job)
     assert not result
-    assert errors == "Sorry, it's too late, letters have already been sent."
+    assert errors == "It's too late to cancel sending, these letters have already been sent."
```
Comment (quis) [inline tests/app/dao/test_jobs_dao.py]:
```suggestion
    assert errors == "It’s too late to cancel sending, these letters have already been sent."
```

**Comment by CrystalPea** [comment]
@quis ahhh correct, thanks! I applied your changes now ☺️

**Pull request merged**